modules/services/avizo.nix

```
@@ -0,0 +1,71 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let cfg = config.services.avizo;
+in {
+
+  meta.maintainers = [ hm.maintainers.pltanton ];
+
+  options.services.avizo = {
+    enable = mkEnableOption ''
+      Whether to enable avizo.
+      Avizo is a simple notification daemon, mainly intended to be used for multimedia keys for example with Sway.
```

This string is interpolated as `Whether to enable {}.`.

```suggestion
      avizo, a simple notification daemon, mainly intended to be used for multimedia keys for example with Sway
```